[terra-toggle] fix update example accessibility issue (#3853)
cerner/terra-core: packages/terra-core-docs/CHANGELOG.md

* Added
  * Added documentation updates for `terra-form-input`.

* Changed

>   * Updated Toggle Button Examples. Now the Interactive elements within the example has dashed border around.

Reviewer: ```suggestion
  * Updated Toggle Button Examples to have a dashed border around interactive elements.

```

Author: Updated. Thanks
